packages/airnode-node/src/reporting/heartbeat.ts

```diff
@@ -56,16 +42,18 @@ export async function reportHeartbeat(state: CoordinatorState): Promise<PendingL
   const httpGatewayUrl = getHttpGatewayUrl(config);
   const httpSignedDataGatewayUrl = getHttpSignedDataGatewayUrl(config);
 
-  const heartbeatPayload = {
+  const timestamp = Date.now();
+
+  const heartbeatPayload = JSON.stringify({
+    timestamp,
```

Comment:
Nit: I'd suggest using the UNIX timestamp (in seconds) and not the JS timestamp which is in ms... But this is a breaking change so we might just not do it.

Reply:
Good point. Unix would be much better imo. And just to confirm, this in in UTC as well?

Why is it a breaking change?

Reply:
Don't you rely on the heartbeat timestamp being in ms? (That's how it worked until now). Changing that to seconds would mean the old values would need to be migrated.

Reply:
ChainAPI hasn't been using the timestamp until now

Reply:
I've made the change by using `Math.round(Date.now())` 👌